.env.example

```
@@ -126,4 +127,4 @@ VITE_ENABLE_MSW=true
 # Production: Your deployed backend URL
 VITE_API_URL=http://localhost:8001
 
-CHROMATIC_PROJECT_TOKEN
\ No newline at end of file
+CHROMATIC_PROJECT_TOKEN
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The CHROMATIC_PROJECT_TOKEN key should be with a value or have an equal sign

(KeyWithoutValue)